html-relative-links/projects/projects.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Projects</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li>
+      <a href="../index.html">Home</a>
+    </li>
+    <li>
+      <a href="../about-me/about-me.html">About Me</a>
+    </li>
+    <li>
+      <a href="../projects/projects.html">Projects</a>
```

Comment:
Whoops! Looks like you are exiting the folder to just go right back into it! In this case, you want the `projects.html` file, which is the same file you are in, which means it is in the same directory, so you can just say the name of the file directly.